namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-prod/resources/irsa.tf

```diff
@@ -0,0 +1,52 @@
+
+module "irsa-api" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=1.0.3"
+
+  namespace        = var.namespace
+  role_policy_arns = [aws_iam_policy.analytical-platform.arn]
+  service_account  = "${var.namespace}-api"
```

Comment:
Out of curiosity, are you using the `-api` suffix to refer the "hmpps-incentives-api" application?